jis-plugins/src/main/java/pl/edu/uj/JImageStream/filters/color/SepiaFilter.java

```diff
@@ -10,7 +10,8 @@ public void apply(int x, int y) {
         int outputRed = (int) ((input.getRed() * .393) + (input.getGreen() * .769) + (input.getBlue() * .189));
         int outputGreen = (int) ((input.getRed() * .349) + (input.getGreen() * .686) + (input.getBlue() * .168));
         int outputBlue = (int) ((input.getRed() * .272) + (input.getGreen() * .534) + (input.getBlue() * .131));
-        setPixel(x, y, new Pixel(Math.min(outputRed, 255), Math.min(outputGreen, 255), Math.min(outputBlue, 255), 255));
+        int alpha = getPixel(x, y).getAlpha();
+        setPixel(x, y, new Pixel(Math.min(outputRed, 255), Math.min(outputGreen, 255), Math.min(outputBlue, 255), alpha));
```

Comment:
we don't ready need to check Math.min anymore in filters since we moved it to setPixel